Directions Example/ViewController.swift

```diff
@@ -18,16 +20,58 @@ class ViewController: UIViewController {
         
         mapView = MGLMapView(frame: view.bounds)
         mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 37.3300, longitude: -122.0312), animated: false)
```

Comment:
could we specify what location we are setting our map to, preferably setting it to a variable?